examples/energy-management-app/energy-management-common/energy-evse/src/EVSEManufacturerImpl.cpp

```diff
@@ -551,6 +552,12 @@ CHIP_ERROR EVSEManufacturer::SendPeriodicEnergyReading(EndpointId aEndpointId, i
     return CHIP_NO_ERROR;
 }
 
+void EVSEManufacturer::UpdateEVFakeReadings(const Amperage_mA maximumChargeCurrent)
+{
+    FakeReadings::GetInstance().SetCurrent(maximumChargeCurrent);
+    FakeReadings::GetInstance().SetPower((FakeReadings::GetInstance().GetVoltage() * maximumChargeCurrent) / 1000);
```

Comment:
could you add a comment on why the division by 1000 ?  I assume this is to make the mA*mV be mW